### ISSUE TITLE
Improve accessibility (using Tab or a screen reader)

### ISSUE THREAD

**Issue by Skystream96** [Codeinwp/visualizer#917]
### Description:
At the moment, our plugin is not quite friendly accessibility wise meaning that you cannot go through it and set up a chart or at least go through the options using Tab or a screen reader.

One user asked about this and indeed they are right as at the moment, you can open the chart library using only Tab and Enter, but from there, you cannot go through the different charts and different options, instead, you go to the other buttons in the dashboard whilst the Visualizer Chart Library Popup remains active.

I'm not sure this would be a huge benefit for the plugin (work needed vs reward) but I think we should consider it for the future in case more users are asking for it.

At the moment I wanted to hear your thoughts on this and then create this feature request and put it on hold until we have more users asking for it.

What do you think @girishpanchal30 @vytisbulkevicius ?

[User asked here.](https://secure.helpscout.net/conversation/1967838676/332142?folderId=212390)
